src/nlayout.rs

```diff
@@ -86,4 +86,28 @@ impl NLayout {
             .map(|i| [i, self.logic_to_phys[i]])
             .collect()
     }
+
+    /// Get physical bit from logical bit
+    fn get_item_logic(&self, logical_bit: usize) -> usize {
+        self.logic_to_phys[logical_bit]
+    }
+
+    /// Get logical bit from physical bit
+    pub fn get_item_phys(&self, physical_bit: usize) -> usize {
+        self.phys_to_logic[physical_bit]
+    }
+
+    /// Swap the specified virtual qubits
+    pub fn swap_logic(&mut self, bit_a: usize, bit_b: usize) {
```

Comment:
My preference would be to rename this to something like, `swap_logical`.

Reply:
Done in: https://github.com/Qiskit/qiskit-terra/pull/7977/commits/9640ed1385c1990437115d23df818726f2656eea